docs/production-readiness/develop.md

```diff
@@ -39,6 +39,7 @@ User code <--> Payload Converter <--> Payload Codec <--> Wire <--> Temporal Serv
 ```
 
 You can run your Payload Codec with a <a class="tdlp" href="/dataconversion#codec-server">Codec Server<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is a Codec Server?</span><br /><br /><span class="tdlppd">A Codec Server is an HTTP server that uses your custom Payload Codec to encode and decode your data remotely through endpoints.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/dataconversion#codec-server">Learn more</a></span></span></a> and use the Codec Server endpoints in Web UI and tctl to decode your encrypted Payload locally.
+For details on how to set up a Codec Server, see (/prod-readiness-context/how-to-set-up-codec-server).
 
 However, if you plan to set up <a class="tdlp" href="/dataconversion#remote-data-encoding">remote data encoding<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is remote data encoding?</span><br /><br /><span class="tdlppd">Remote data encding is using your custom Data Converter to decode (and encode) your Payloads remotely through endpoints.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/dataconversion#remote-data-encoding">Learn more</a></span></span></a> for your data, ensure that you consider all security implications of running encryption remotely before implementing it.
```

Comment:
While it's technically possible for workers to use a Codec Server for their encoding we don't recommend it. I would reword this to avoid mention of encoding.